code/game/turfs/simulated/floor/f13_floors.dm

```diff
@@ -7,22 +7,16 @@
 */
 
 /turf/open/floor/plating/f13/outside
-	icon = 'icons/turf/f13desert.dmi'
+	name = "\proper desert"
 	icon_state = "wasteland1"
+	icon = 'icons/turf/f13desert.dmi'
 	light_range = 3
 	light_power = 0.75
-	planetary_atmos = TRUE
 
-/turf/open/floor/plating/f13
+/turf/open/floor/plating/f13 // don't use this for anything, /f13/ is essentially just the new /unsimulated/ but for planets and should probably be phased out entirely everywhere
 	gender = PLURAL
-	name = "\proper desert"
```

Comment:
name it to instead: `name = "What the fuck mappers? why is this here"`
desc: `description = "If found, scream at the github repo about this`"